websites/W/WaniKani/presence.ts

```diff
@@ -30,12 +30,11 @@ presence.on("UpdateData", () => {
 						".lessons-and-reviews__button"
 					);
 					if (buttons.length === 2) {
-						const lessons: number =
-								+buttons[0].querySelector("span").textContent,
-							reviews: number = +buttons[1].querySelector("span").textContent;
+						const lessons = +buttons[0].querySelector("span").textContent,
+							reviews = +buttons[1].querySelector("span").textContent;
 						details = "Viewing Dashboard";
 						state = `${lessons} lessons | ${reviews} reviews`;
-						smallImageText = document.querySelector<HTMLAnchorElement>(
```

Comment:
Why was the type removed?

Reply:
Didn't feel that it was necessary, as I'm only accessing `.textContent`. Would you prefer I add it back?

Reply:
Given we’re using typescript, any added types are preferable